packages/manager/apps/pci-load-balancer/src/components/listing/OperatingStatus.component.tsx

```diff
@@ -0,0 +1,44 @@
+import { useTranslation } from 'react-i18next';
+import { OsdsChip } from '@ovhcloud/ods-components/react';
+import { useMemo } from 'react';
+import { OdsChipAttribute } from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+
+type OperatingStatusComponentProps = {
+  operatingStatus: string;
+};
+
+export default function OperatingStatusComponent({
+  operatingStatus,
```

Comment:
operationStatus possible values are known, an enum or a type maybe instead of simple string?